test/validation/api/queue/queue.c

```diff
@@ -326,6 +414,7 @@ void queue_test_info(void)
 odp_testinfo_t queue_suite[] = {
 	ODP_TEST_INFO(queue_test_capa),
 	ODP_TEST_INFO(queue_test_mode),
+	ODP_TEST_INFO(queue_test_lockfree),
```

Comment:
Should there be a corresponding `queue_test_waitfree`?

Reply:
Not yet, since I don't have an implementation for that yet. There's no code to test the test.